Add warning for override entities to set table to the correct name (#580)
sulu/sulu-docs: cookbook/extend-entities.rst

    use Sulu\Bundle\SecurityBundle\Entity\User as SuluUser;

    /**

>      * Following annotations are required and should not be changed:

Reviewer: ```suggestion
     * The following annotations are required for replacing the table of the extended entity:
```

.. warning::

    Your Entity can have own properties, but they should have at least default values.
    Otherwise the normal features of Sulu could crash (like the
    `sulu:security:user:create` command).

>     The `@ORM\\Table` annotation need to match the exist table else you will get strange
>     errors when query data from it.

Reviewer: ```suggestion
.. warning::

    The `@ORM\\Table` annotation on your entity must match the table of the extended entity. 
    Otherwise, doctrine might run into errors when querying data of the entity.
```

.. warning::

>     If you override the entities you should provide a data migration script to avoid data lose.

Reviewer: ```suggestion
    If you override entities in an existing project, you need to migrate the existing data to avoid data loss.
```
